client/starwhale/utils/config.py

```diff
@@ -147,8 +147,13 @@ def current_instance(self) -> str:
         return str(self._config["current_instance"])
 
     def get_sw_instance_config(self, instance: str) -> t.Dict[str, t.Any]:
-        instance = self._get_instance_alias(instance)
-        return self._config["instances"].get(instance) or {}
+        instance_alias = self._get_instance_alias(instance)
```

Comment:
`_get_instance_alias` should throw an exception when getting duplicate urls?

Reply:
done